bin/etest

```diff
@@ -230,77 +230,59 @@ die_handler()
     exit ${rc}
 }
 
-# Returns success if there are no stale processes remaining in the cgroup for this test and failure if there are any.
-no_process_leaks_remain()
+assert_no_process_leaks()
```

Comment:
We used to have two functions -- `no_process_leaks_remain` and `assert_no_process_leaks` with very similar code in them. I still need a separate callable function to check for process leaks inside `assert_no_process_leaks` but I decided to make it a super small one-liner private function to simplify things a bit.